README.md

```diff
@@ -1,6 +1,26 @@
 # Inconsolata
 
-Open-source Monospace Font for code listings by Raph Levien
+Open-source monospace font for code listings, originally by [**@raphlinus**](https://github.com/raphlinus/)
```

Comment:
This sentence is already in the description, so I added a link to his profile here.